examples/demo12.py

```diff
@@ -78,7 +83,7 @@ def main(argv):
     SED_names = ['CWW_E_ext', 'CWW_Sbc_ext', 'CWW_Scd_ext', 'CWW_Im_ext']
     SEDs = {}
     for SED_name in SED_names:
-        SED_filename = os.path.join(galsim.meta_data.share_dir, '{0}.sed'.format(SED_name))
+        SED_filename = os.path.join(datapath, 'SEDs/{0}.sed'.format(SED_name))
```

Comment:
>I like the better organization of putting the SEDs and bandpasses each in their own subdirectory of the main `share_dir`.

Thanks!  me too.

>we should at the very least declare this as an API change

added to CHANGELOG

>it might be worth either checking if the path looks like "..../share/galsim/*.sed" and adding in the "SEDs" bit with a deprecation warning

Done.

>Second, I think it would be nice if the Bandpass and SED constructors could look in share_dir/bandpasses and share_dir/SEDs automatically

Done.

>This should really be join(datapath, 'SEDs', '{0}.sed'.format(SED_name).

Also done.  I fixed the Bandpass and SED occurrences of this.  There were a few more that only live in the tests dir that I didn't bother with.